### PR TITLE
Chukwuemeka/user registration and login

### PR DESCRIPTION
**Pull request by mxsurft**
Registration and Login features created.

### CODE REVIEW

**Comment by mxsurft** [comment]
[User Login - Part 1](https://trello.com/c/bUbsvcju/3-user-login-part-1)

**Comment by mxsurft** [comment]
[User Registration - Part 1](https://trello.com/c/IAhsD7nH/2-user-registration-part-1)

**Review by adelaykay — CHANGES_REQUESTED** [review]
1. Remove Console Logs
2. Rename token property in json response to accessToken
3. Remove unused mongoose import in authentication.js

**Comment by mxsurft** [comment]
1. Remove Console Logs - **Done**
2. Rename token property in json response to accessToken - - **Done**
3. Remove unused mongoose import in authentication.js - **Done**

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ If you want to contribute to this project, you are welcome to do so. Here are so
 * Suggest new features or enhancements using the feature request template.
 * Share this project with your friends or colleagues who might find it useful or interesting.
 
+### Note
+In the User model a createJWT method was created, which creates a token for a user. In the method, the following variables from the env were used JWT_SECRET and JWT_LIFETIME.
+Endeavour to create these variables in your own env file and assign appropriate values.
+Also create a MONGO_URI variable in your env file and assign an appropriate database string.
```
Comment (adelaykay) [inline README.md:29]: Awesome, awesome. 😉

**Review by adelaykay — APPROVED** [review]

**Review by Omoefe-Ugboma — APPROVED** [review]

**Review by Omoefe-Ugboma — APPROVED** [review]

**Pull request merged**